dace/codegen/targets/cpu.py

```diff
@@ -1,12 +1,15 @@
 import itertools
 import warnings
 
-from dace import data, registry, memlet as mm
+import dace
+from dace import data, dtypes, registry, memlet, subsets, symbolic
```

Comment:
also, `from dace import memlet` might be problematic due to shadowing any variable called `memlet`